spec/features/new_results_page_filters/subjects_spec.rb

```diff
@@ -0,0 +1,329 @@
+require 'rails_helper'
+
+RSpec.feature 'Results page new subject filter' do
+  include StubbedRequests::Courses
+  include StubbedRequests::SubjectAreas
+  include StubbedRequests::Subjects
+
+  let(:filter_page) { PageObjects::Page::ResultFilters::SubjectPage.new }
+  let(:results_page) { PageObjects::Page::ResultsWithNewFilters.new }
+  let(:base_parameters) { results_page_parameters }
+
+  before do
+    activate_feature(:new_filters)
+
+    stub_courses(query: base_parameters, course_count: 10)
+    stub_subject_areas
+    stub_subjects
+  end
+
+  describe 'applying a filter' do
+    before do
+      stub_courses(query: base_parameters, course_count: 10)
+    end
+
+    context 'with less than 4 subjects selected' do
+      before do
+        stub_courses(
+          query: base_parameters.merge('filter[subjects]' => '00,F1'),
+          course_count: 10,
+        )
+      end
+
+      it 'lists the results' do
+        results_page.load
+        results_page.subjects_filter.link.click
+
+        expect(filter_page.heading.text).to eq(I18n.t('page_titles.subjects_filter'))
+        filter_page.subject_areas.first.subjects[0].checkbox.click
+        filter_page.subject_areas.second.subjects[3].checkbox.click
+
+        filter_page.continue.click
+
+        expect(results_page.heading.text).to eq('Teacher training courses')
+        expect(results_page.subjects_filter.subjects.first.text).to eq('Chemistry, Primary')
+
+        expect(results_page.courses.count).to eq(10)
+      end
+    end
+
+    context 'with subjects selected' do
```

Comment:
I might be missing something, but this and the above test seem to be testing the same thing.

Reply:
Yes agree! Have removed the dupped spec